src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import "./Navbar.scss"
+
+
+function Navbar() {
+  return (
+    <div className="Navbar">
+         <div className="navbar__left">
+            <div className="navbar__icon"><img alt="Logo" src="assets/skillChain.svg"/></div>
```

Comment:
for some reason the code does not work in mine. 
so I cant say for sure this part works bt documentation says to import the img like
import logo from './../../Assets/SkillChain.svg'

<img alt="Logo" src={logo}/>

and folder Assets should be assets i think

Reply:
Thank you